tools/ratbagctl.in

```diff
@@ -279,11 +287,11 @@ def print_device(d, level):
         pass
 
     print(" " * level + "{} - {}".format(d.id, d.name))
-    print(" " * level + "               SVG: {}".format(svg_name))
-    print(" " * level + "      Capabilities: {}".format(", ".join(capabilities)))
-    print(" " * level + " Number of Buttons: {}".format(len(p.buttons)))
-    print(" " * level + "    Number of Leds: {}".format(len(p.leds)))
-    print(" " * level + "Number of Profiles: {}".format(len(d.profiles)))
+    print(" " * level + "                 SVG: {}".format(svg_name))
+    print(" " * level + "        Capabilities: {}".format(", ".join(capabilities)))
+    print(" " * level + "   Number of Buttons: {}".format(len(p.buttons)))
+    print(" " * level + "      Number of Leds: {}".format(len(p.leds)))
+    print(" " * level + "  Number of Profiles: {}".format(len(d.profiles)))
```

Comment:
Nitpick: aren't you missing a space here if the goal is to right-align the labels?

Reply:
huh, this is rubbish anyway, it's a leftover from an earlier version. I'll drop this bit, thanks